srcpkgs/speed-dreams/patches/musl.diff

```diff
@@ -0,0 +1,12 @@
+--- a/src/modules/simu/simuv2.1/SOLID-2.0/src/C-api.cpp
++++ b/src/modules/simu/simuv2.1/SOLID-2.0/src/C-api.cpp
+@@ -60,9 +60,7 @@
+ typedef map<DtObjectRef, Object *> ObjectList;
+ typedef set<Encounter> ProxList;
+ 
+-#if defined( WIN32) || (__APPLE__)
+ #define uint unsigned int
+-#endif
+ 
+ PointBuf pointBuf;
+ IndexBuf indexBuf;
```

Comment:
this file should probably be named musl.patch, not musl.diff

Reply:
Aren't diff and patch synonyms? (I mean the files, not the programs). ".diff" is my personal preference, but I'll rename it if that's preferred to keep consistency.

Reply:
I think we don't really use `diff` in the repo, even if it's supported.